src/app/resources/resources.tpl.html

```diff
@@ -43,13 +43,16 @@ <h2 class="raml-console-resource-section-title">
       <li id="{{generateId(resource.pathSegments)}}" class="raml-console-resource-list-item" ng-repeat="resourceGroup in resourceGroups" ng-init="resource = resourceGroup[0]">
         <resource-group
           base-uri="baseUri"
-          base-uri-parameters="raml.baseUriParameters"
+          base-uri-parameters="baseUriParameters"
           first-resource="resource"
           generate-id="generateId"
           index="$index"
           protocols="protocols"
+          resources-collapsed="resourcesCollapsed"
           resource-group="resourceGroup"
-          resource-list="resourceList"></resource-group>
+          resource-list="resourceList"
+          single-view="singleView">
```

Comment:
This is weird. Either property name should be something like `viewType` or you should just use the attribute with no arguments (or `true`).